src/test/ui/rfc-2632-const-trait-impl/without-tilde.rs

```diff
@@ -3,4 +3,5 @@
 #![feature(const_trait_impl)]
 
 struct S<T: const Tr>;
```

Comment:
Hm, loss of `~` seems wrong here.

Reply:
Sorry for taking so long to get back to this. Can you investigate/fix the loss of the `~` token here?

Reply:
I'm sorry for taking so long to return to this too 😅 

The `~` got missing here because `const` was reinterpreted as a keyword. I've pushed a commit with recovery for that test, so it's better now.